packages/popmotion-pose/src/dom/unit-conversion.ts

```diff
@@ -78,7 +85,6 @@ export const convertPositionalUnits = (state: PoserState, pose: Pose) => {
   elementStyler.render();
 
   const targetBbox = element.getBoundingClientRect();
-  const newPose = { ...pose };
```

Comment:
`pose` was already a copy, so there is no need to copy it again, https://github.com/Popmotion/popmotion/pull/571/files#diff-6575bee1d0a5453bf870f9c62efe8823R83